tests/main/snap-run-gdbserver/task.yaml

```diff
@@ -33,5 +33,5 @@ execute: |
     echo c | eval $gdb_cmd
 
     # ensure the program ran and was running as a user
-    MATCH hello-hello-hello < stdout
+    retry --wait 1 -n 5 sh -c 'MATCH hello-hello-hello < stdout'
```

Comment:
Can we instead `wait` for the background snap run ... job to finish?

Reply:
I think that retry is ok here. `wait` will wait indefinitely for command to complete. Alternative could be to use something like `timeout 10s wait`, but then `wait` is a builtin shell command so we can't really call it like that.